manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -378,13 +378,13 @@ private function getDatabaseUrl(ContainerBuilder $container): string
         $dbName = '';
 
         if ($name = $container->getParameter('database_name')) {
-            $dbName = '/'.$name;
+            $dbName = '/'.$this->encodeUrlParameter($name);
         }
 
         return sprintf(
             'mysql://%s%s:%s%s',
             $userPassword,
-            $container->getParameter('database_host'),
+            $this->encodeUrlParameter($container->getParameter('database_host')),
```

Comment:
Is it really necessary to encode he hostname? It should be valid already, because, well, it is a hostname. 😄

Reply:
No, that was unintentional.